src/main/kotlin/com/autonomousapps/internal/advice/AdvicePrinter.kt

```diff
@@ -10,24 +10,40 @@ internal class AdvicePrinter(
   private val dependencyMap: (String) -> String = { it },
 ) {
 
-  fun line(configuration: String, printableIdentifier: String, was: String = ""): String = when (dslKind) {
-    DslKind.KOTLIN -> "  $configuration($printableIdentifier)$was"
-    DslKind.GROOVY -> "  $configuration $printableIdentifier$was"
-  }
+  fun line(configuration: String, printableIdentifier: String, was: String = ""): String =
+    "  $configuration$printableIdentifier$was"
```

Comment:
Then `when` is now done in `fun gav(coordinates: Coordinates)` already (see changes below).

This is because we might need to add a `{ ... }` configuration block as part of the "printableIdentifier" for a `requireCapability`. Because of that, we need to make the decision if we want to add `(` and `)` there (as `)` might be in the middle of the string before the `{`).